src/accelerate/components/List.tsx

```diff
@@ -73,6 +81,14 @@ export default function List( props: Props ) {
 							onChange={ e => {
 								setPage( 1 );
 								setUser( parseInt( e.target.value, 10 ) );
+								analytics.track(
+									__( 'filter', 'altis-analytics' ),
+									{
+										location: __( 'dashboard', 'altis-analytics' ),
+										filter_type: __( 'author', 'altis-analytics' ),
+										filter_value: e.target.options[e.target.options.selectedIndex].text,
```

Comment:
Again, is this the right value to be using here?

Reply:
So currently the filter only allows the select of `All Authors` and `My Content`. The respect values are ` ` and `1` which I figured would have no real value so I thought it was best to use the option select. Potentially a better solution to use?

Reply:
This should be  using `e.target.value` as the `filter_value` value